utilities/trajectory_generator.py

```diff
@@ -1,21 +1,41 @@
-from wpimath.geometry import Pose2d
+from wpimath.geometry import Pose2d, Rotation2d, Translation2d
 from typing import List
 
 from utilities.scalers import scale_value
 
 
+field_x = 324 * 0.0254  # half field x dimension
+field_y = 162 * 0.0254  # half field y dimension
+FIELD_TRANFORM = Translation2d(field_x, field_y)
```

Comment:
This is no longer a transform, so should not be called as such.

Should this be called `GOAL_POSITION` or similar?